apps/search/test/marionette/search_rocketbar_test.js

```diff
@@ -100,8 +110,18 @@ marionette('Search - Rocketbar Test', function() {
     assert.ok(!rocketbar.clear.displayed());
 
     // Perform a search
-    rocketbar.enterText('a test\uE006');
-    rocketbar.switchToBrowserFrame(server.url('sample.html'));
+    rocketbar.enterText(textToSearch, true);
+    rocketbar.switchToBrowserFrame(searchTemplate);
+  });
+
+  test('Browser search', function() {
+    home.waitForLaunch();
+    rocketbar.homescreenFocus();
+    var textToSearch = 'a test';
+    rocketbar.enterText(textToSearch, true);
```

Comment:
Mhm, I don't think we always want to switch to the browser frame, but +1 to rephrase this to show user's intent.